examples/take_pic_with_csc_with_canon_camera.py

```diff
@@ -0,0 +1,39 @@
+import asyncio
+from lsst.ts import salobj
+
+
+async def main():
+    print("main method")
+    domain = salobj.Domain()
```

Comment:
you probably need to "Press CTRL-C now a few times to exit." because of the way salobj is handling the DDS read threads. The safest way would be to encapsulate the code in a `with` statement.

instead of doing

```
domain = salobj.Domain()
...
```

do 

```
async with salobj.Domain() as domain:
    ....
```

Then the code will exit at the end without the need to Ctrl+c. :)

Reply:
Thanks! I modified the script.